js/reduce.algo2/docs/variables.json

```
@@ -1,4 +1,284 @@
 [
+    {
+        "source": "ccsf",
+        "name": "date_ccsf",
+        "description": "Date de début de la procédure CCSF",
+        "type": "string",
+        "format": "date-time",
+        "computed": true
+    },
+    {
+        "source": "compte",
+        "name": "compte_urssaf",
+        "description": "Compte administratif URSSAF",
+        "type": "string",
+        "computed": true
+    },
+    {
+        "source": "cotisation",
+        "name": "cotisation_moy12m",
+        "description": "Montant moyen de cotisations calculé sur 12 mois consécutifs.",
+        "type": "number",
+        "computed": true
+    },
+    {
+        "source": "cotisation",
+        "name": "ratio_dette",
+        "description": "ratio_dette divise montant_part_ouvriere et montant_part_patronale par cotisation_moy12m.",
```

`ratio_dette = (montant_part_ouvriere + montant_part_patronale) / cotisation_moy12m`

Privilegier les formules au texte quand c'est un calcul simple

---

Ok ! N'hésite pas à me pinger sur les autres descriptions que tu préfererais exprimer sous forme de formules.

```suggestion
        "description": "ratio_dette = (montant_part_ouvriere + montant_part_patronale) / cotisation_moy12m",
```